src/applications/travel-pay/tests/containers/TravelPayStatusApp.unit.spec.jsx

```diff
@@ -224,4 +225,26 @@ describe('App', () => {
       ).to.eq(`${date} at ${time} appointment`);
     });
   });
+
+  it('renders pagination correctly', async () => {
```

Comment:
I'd love to see `expect`s for how pagination interacts with sorting:

1. Default sort order, then: Claim appts on page 2 occurred after claim appts on page 1
2. Sort by oldest first, then: Claim appts on page 2 occurred before claim appts on page  1

Reply:
Let's also add a test for showing only 10 claims on a page.

Reply:
Agreed - everything else looks good to me!

Reply:
There's no easy way to do this with the constraints of using the shadow DOM with react testing library 😞 Unfortunately I think we'll have to add those kinds of assertions to our cypress tests for now